MoreLinq/ExceptAll.cs

```diff
@@ -0,0 +1,186 @@
+﻿#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+namespace MoreLinq {
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    static partial class MoreEnumerable {
+
+        /// <summary>
+        /// Returns the sequence of elements in the first sequence which aren't
+        /// in the second sequence.
+        /// </summary>
+        /// <remarks>
+        /// This is a sequence operation; if multiple elements in <paramref name="first"/> are equal, 
```

Comment:
I would add a note here that this is unlike `Except` from LINQ (I was wondering why we wanted `ExceptAll`...